### ISSUE TITLE
_event$touches$ is undefined

### ISSUE THREAD

**Issue by kamilmoskal** [atlassian/react-beautiful-dnd#1292]
Idk if it's a bug or i messed up something with my code, but 99% was copied from tutorial. I can;t find similar problem so i decide to post it here.

### Actual behavior

i did video tutorial how to set up basic draggable list and everything looks and works fine on chrome and firefox. But when i enabled mobile mode in firefox in dev tools (in chrome mobile mode dev tools working fine), and trying to move dragged list item (through 1s press),  item is not moving and i getting this errors:

react-beautiful-dnd

An error has occurred while a drag is occurring.
Any existing drag will be cancelled.

> TypeError: _event$touches$ is undefined

👷‍ This is a development only message. It will be removed in production builds.

raw 
error { target: Window, isTrusted: true, message: "TypeError: _event$touches$ is undefined", filename: "http://localhost:3000/static/js/0.chunk.js", lineno: 21964, colno: 13, error: TypeError, srcElement: Window, currentTarget: Window, eventPhase: 2, … }
index.js:1437

TypeError: _event$touches$ is undefined

### What version of `React` are you using?

   "react": "^16.8.6",

### What version of `react-beautiful-dnd` are you running?

 "react-beautiful-dnd": "^11.0.2",

### What browser are you using?

Firefox 66.0.3

**Comment by alexreardon** [comment]
I tried it out in firefox and I also found this error. I am not sure what is causing it, but i can look deeper

**Comment by alexreardon** [comment]
when using breakpoints everything seems fine..

Perhaps it is a transpilation issue

**Comment by alexreardon** [comment]
```js
var _event$touches$ = event.touches[0],
            clientX = _event$touches$.clientX,
            clientY = _event$touches$.clientY;
```

It looks like this assignment is causing issues. It might be a babel issue

**Comment by alexreardon** [comment]
Okay, the assignment statement itself is fine. It looks like sometimes in firefox event.touches for a touchmove is empty. Setting a breakpoint causes it always to be populated. I think this is a firefox regression. Would could add a little defense for this...

**Comment by alexreardon** [comment]
I'll keep looking

**Comment by alexreardon** [comment]
It turns out this is a bug in firefox. I created a [bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1549868)

I have shipped a workaround in `11.0.3` so that touch dragging will work again for firefox
